yaas/parsers/bootstrap.py

```diff
@@ -0,0 +1,104 @@
+# coding: utf-8
+
+from __future__ import absolute_import
+from __future__ import print_function
+
+import getpass
+import inspect
+
+from ..bootstrap import Bootstrap
+
+
+def command(parent_parser):
+    parser = parent_parser.add_parser(
+        'bootstrap',
+        help=inspect.getdoc(Bootstrap))
+
+    subcommands = [
+        bootstrap_agents,
+        bootstrap_show,
+        ]
+
+    subparsers = parser.add_subparsers()
+    for subcommand in subcommands:
+        subcommand(subparsers)
+
+
+def bootstrap_agents(parent_parser):
+
+    """ Install and configure ambari-agent on specified hosts. """
+
+    def agents(client, args):
+        with open(args.ssh_key, 'r') as f:
+            ssh_key = f.read()
+        res = client.bootstrap.agents(
+            hosts=args.hosts,
+            sshKey=ssh_key,
+            user=args.user,
+            userRunAs=args.userRunAs)
+        if not client.raw:
```

Comment:
Doh, but we don't want to disable anyone from using some other task-tracking tool. I suppose its inconsequential, though, since `yaas task` will be in the first release (right?). Thanks btw :)